lightkurve/factory.py

```diff
@@ -275,6 +281,154 @@ def _make_aperture_extension(self):
         hdu.header['EXTNAME'] = 'APERTURE'
         return hdu
 
+
+    @staticmethod
+    def from_fits_images(images, position, size=(11, 11), extension=1,
```

Comment:
I think this should now be a standalone function rather than a `@staticmethod`.

`TargetPixelFileFactory.from_fits_images()` does not make much sense because the method returns a `TargetPixelFile` rather than a `TargetPixelFileFactory`.  (And, in general, people find static methods confusing.)